nms/app/packages/magmalte/app/views/equipment/EnodebDetailSummaryStatus.js

```diff
@@ -60,16 +67,18 @@ export function EnodebStatus() {
       },
       {
         category: 'Transmit Enabled',
-        value: enbInfo.enb.config.transmit_enabled ? 'Enabled' : 'Disabled',
+        value: enbInfo.enb.enodeb_config?.managed_config?.transmit_enabled
+          ? 'Enabled'
+          : 'Disabled',
         statusCircle: true,
-        status: enbInfo.enb.config.transmit_enabled,
+        status: enbInfo.enb.enodeb_config?.managed_config?.transmit_enabled,
         tooltip: 'current transmit configuration on the eNodeB',
       },
```

Comment:
We need some type of `N/A` value for an unmanaged eNB, I don't think `Enabled/Disabled` captures this well.

Reply:
May be the diff is confusing here. This is for transmit being enabled or disabled.
For "externally managed eNodeBs" the value displayed is either "True" or "False"
  const managedConfig: DataRows[] = [
    [
      {
        category: 'eNodeB Externally Managed',
        value: 'False',
      },
    ],